src/timeMachine/components/RawFluxDataTable.tsx

```diff
@@ -1,106 +1,75 @@
 // Libraries
-import React, {PureComponent} from 'react'
-import memoizeOne from 'memoize-one'
+import React, {FunctionComponent, useState} from 'react'
 import RawFluxDataGrid from 'src/timeMachine/components/RawFluxDataGrid'
 import {FromFluxResult} from '@influxdata/giraffe'
 
 // Utils
 import {
-  parseFiles,
-  parseFilesWithObjects,
-  parseFilesWithFromFlux,
   parseFromFluxResults,
 } from 'src/timeMachine/utils/rawFluxDataTable'
-import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import {DapperScrollbars, FusionScrollEvent} from '@influxdata/clockface'
 
 interface Props {
-  files?: string[]
-  parsedResults?: FromFluxResult
+  parsedResults: FromFluxResult
   width: number
   height: number
   disableVerticalScrolling?: boolean
   startRow?: number
 }
 
-interface State {
-  scrollLeft: number
-  scrollTop: number
-}
-
-class RawFluxDataTable extends PureComponent<Props, State> {
-  public state = {scrollLeft: 0, scrollTop: 0}
-
-  private parseFilesWithFromFlux = memoizeOne(parseFilesWithFromFlux)
-  private parseFiles = memoizeOne(parseFiles)
-  private parseFilesWithObjects = memoizeOne(parseFilesWithObjects)
-
-  public render() {
-    const {
-      width,
-      height,
-      files,
-      disableVerticalScrolling,
-      startRow,
-    } = this.props
-
-    const {scrollTop, scrollLeft} = this.state
+const RawFluxDataTable: FunctionComponent<Props> = ({
+  width,
+  height,
+  parsedResults,
+  disableVerticalScrolling,
+  startRow
+}) => {
+  const [scrollLeft, setScrollLeft] = useState(0);
+  const [scrollTop, setScrollTop] = useState(0);
+  const onScrollbarsScroll = (e: FusionScrollEvent) => {
+    const {scrollTop: sTop, scrollLeft: sLeft} = e
+    setScrollLeft(sLeft)
+    setScrollTop(sTop)
+  }
 
-    const tableWidth = width
-    const tableHeight = height
+  const tableWidth = width
+  const tableHeight = height
 
-    const {parsedResults} = this.props
-    let data
-    let maxColumnCount
-    let parseFunction = this.parseFiles
-    if (parsedResults) {
-      const parsed = parseFromFluxResults(parsedResults)
-      data = parsed.tableData
-      maxColumnCount = parsed.max
-    } else {
-      if (isFlagEnabled('parseObjectsInCSV')) {
-        parseFunction = this.parseFilesWithObjects
-      }
-      parseFunction = this.parseFilesWithFromFlux
-      const parsed = parseFunction(files)
-      data = parsed.data
-      maxColumnCount = parsed.maxColumnCount
-    }
```

Comment:
We removed this `else` since it's an outdated view of the RawFluxDataTable as a component that parses data (which is happening in the respective parent)